build/dependencies.props

```diff
@@ -1,4 +1,4 @@
-﻿<Project>
+<Project>
```

Comment:
Revert?

Reply:
These diffs happen all the time on props/csproj files. I don't bother reverting them most of the time. If you care enough, I'll revert it.

Reply:
Rule of thumb is PRs should not contain any unintended changes.  Recommend instead figuring out why these diffs are happening (maybe file encoding?) and how to fix the issue long term.

Reply:
This is getting inserted: https://en.wikipedia.org/wiki/Byte_order_mark